core/pom.xml

```diff
@@ -155,6 +151,71 @@ limitations under the License.
 
   <profiles>
     <profile>
+      <id>it</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>failsafe-integration-test</id>
+                <phase/>
```

Comment:
Makes sense. I'll add a comment. I'm working up a pull request which fixes several issues, so you can fix the comment when you review.